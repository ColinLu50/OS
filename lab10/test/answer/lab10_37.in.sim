1706
122
190
104
190
100
