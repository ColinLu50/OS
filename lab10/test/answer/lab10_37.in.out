FCFS
86 -> 92 -> 50 -> 0 -> 70 -> 25 -> 42 -> 61 -> 68 -> 61 -> 90 -> 41 -> 39 -> 36 -> 27 -> 15 -> 13 -> 63 -> 104 -> 50 -> 7 -> 71 -> 95 -> 99 -> 58 -> 47 -> 94 -> 6 -> 18 -> 73 -> 3 -> 11 -> 25 -> 91 -> 45 -> 50 -> 17 -> 32 -> 28 -> 53 -> 17 -> 91 -> 39 -> 51 -> 77 -> 33 -> 3 -> 17 -> 102 -> 16 -> 74
1706
SSTF
86 -> 90 -> 91 -> 91 -> 92 -> 94 -> 95 -> 99 -> 102 -> 104 -> 77 -> 74 -> 73 -> 71 -> 70 -> 68 -> 63 -> 61 -> 61 -> 58 -> 53 -> 51 -> 50 -> 50 -> 50 -> 47 -> 45 -> 42 -> 41 -> 39 -> 39 -> 36 -> 33 -> 32 -> 28 -> 27 -> 25 -> 25 -> 18 -> 17 -> 17 -> 17 -> 16 -> 15 -> 13 -> 11 -> 7 -> 6 -> 3 -> 3 -> 0
122
SCAN
86 -> 77 -> 74 -> 73 -> 71 -> 70 -> 68 -> 63 -> 61 -> 58 -> 53 -> 51 -> 50 -> 47 -> 45 -> 42 -> 41 -> 39 -> 36 -> 33 -> 32 -> 28 -> 27 -> 25 -> 18 -> 17 -> 16 -> 15 -> 13 -> 11 -> 7 -> 6 -> 3 -> 0 -> 0 -> 90 -> 91 -> 92 -> 94 -> 95 -> 99 -> 102 -> 104
190
C-SCAN
86 -> 77 -> 74 -> 73 -> 71 -> 70 -> 68 -> 63 -> 61 -> 58 -> 53 -> 51 -> 50 -> 47 -> 45 -> 42 -> 41 -> 39 -> 36 -> 33 -> 32 -> 28 -> 27 -> 25 -> 18 -> 17 -> 16 -> 15 -> 13 -> 11 -> 7 -> 6 -> 3 -> 0 -> 0 -> 108 -> 104 -> 102 -> 99 -> 95 -> 94 -> 92 -> 91 -> 90
104
LOOK
86 -> 77 -> 74 -> 73 -> 71 -> 70 -> 68 -> 63 -> 61 -> 58 -> 53 -> 51 -> 50 -> 47 -> 45 -> 42 -> 41 -> 39 -> 36 -> 33 -> 32 -> 28 -> 27 -> 25 -> 18 -> 17 -> 16 -> 15 -> 13 -> 11 -> 7 -> 6 -> 3 -> 0 -> 90 -> 91 -> 92 -> 94 -> 95 -> 99 -> 102 -> 104
190
C-LOOK
86 -> 77 -> 74 -> 73 -> 71 -> 70 -> 68 -> 63 -> 61 -> 58 -> 53 -> 51 -> 50 -> 47 -> 45 -> 42 -> 41 -> 39 -> 36 -> 33 -> 32 -> 28 -> 27 -> 25 -> 18 -> 17 -> 16 -> 15 -> 13 -> 11 -> 7 -> 6 -> 3 -> 0 -> 104 -> 102 -> 99 -> 95 -> 94 -> 92 -> 91 -> 90
100
